biome.json

```diff
@@ -38,7 +38,7 @@
         "noPrecisionLoss": "off",
         "noSwitchDeclarations": "off",
         "noChildrenProp": "off",
-        "noUnusedVariables": "error"
+        "noUnusedVariables": "warn"
```

Comment:
sometimes erroneous code and unused code go in the same line and its hard to distinguish which is which.`warn` is underlined yellow and error is underline in red
<img width="239" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/d54992c3-8e22-4d3a-83f7-7b3b832406bd">